### PR TITLE
docs: Q1 2024 Roadmap

### PR DESCRIPTION
**Pull request by wsxiaoys**

### CODE REVIEW

**Comment by wsxiaoys** [comment]
cc @gyxlucy for some comments

File: website/docs/roadmap.md [hunk 1]
```diff
@@ -2,8 +2,20 @@
 
 We continuously work on updating our roadmap and we love to discuss those with our community. Feel encouraged to participate.
 
+## Q1 2024
+
+* 🎊 Supports embedding api for document context related use cases: https://github.com/TabbyML/tabby/issues/790
+* 🔐 SSO / OAuth support in Tabby EE: https://github.com/TabbyML/tabby/issues/1039
```
Comment (gyxlucy) [inline website/docs/roadmap.md:8]: How about `authorization` for EE? Like owner/admin/member access permissions. Feasible for Q1?

File: website/docs/roadmap.md [hunk 2]
```diff
@@ -2,8 +2,20 @@
 
 We continuously work on updating our roadmap and we love to discuss those with our community. Feel encouraged to participate.
 
+## Q1 2024
+
+* 🎊 Supports embedding api for document context related use cases: https://github.com/TabbyML/tabby/issues/790
```
Comment (gyxlucy) [inline website/docs/roadmap.md:7]: Do we consider integrating this API into chat playground in Q1 as well? Once the API is ready, I suppose the integrations should be rather straightforward.

File: website/docs/roadmap.md (hunk 1 above)
Comment (wsxiaoys) [reply website/docs/roadmap.md:8]: should be feasible - so far we only have two tiers of authorization (admin / non-admin)

File: website/docs/roadmap.md (hunk 2 above)
Comment (wsxiaoys) [reply website/docs/roadmap.md:7]: This one is a bit tricky - it's still not clear how these context should be integrated.

One nice PoC built by sourcegraph folks is https://opencodegraph.org/ - I feel at least we could do some integration with our code browser (once it's implemented in #538).

The code browser could use some chat ui, probably.

Comment (gyxlucy) [reply website/docs/roadmap.md:7]: Cool let's research and discuss more in Q1 then.

**Review by icycodes — APPROVED** [review]

**Pull request merged**